Crucible.NPCs/CrucibleNpcCalendarVisitConfig.cs

```diff
@@ -43,11 +42,10 @@ public void AddToDay(CrucibleNpcTemplate npcTemplate)
             }
 
             // TODO: Allow adding to start / middle / end / before npc / after npc
-            CrucibleDayManager.BackfillDaysTo(this.Day);
-            var count = CrucibleDayManager.GetNpcTemplatesForDay(this.Day).Count();
-            CrucibleDayManager.InsertNpcTemplateToDay(this.Day, count, npcTemplate);
+            //CrucibleDayManager.BackfillDaysTo(this.Day);
+            //var count = CrucibleDayManager.GetNpcTemplatesForDay(this.Day).Count();
+            //CrucibleDayManager.InsertNpcTemplateToDay(this.Day, count, npcTemplate);
         }
```

Comment:
There were several places I removed the disabled regions which were not required for 0.5.0 compatibility. Let me know and I can revert these but I am planning on expanding them as my next project to allow for custom NPCs within Crucible. So I figured I would leave this in.